File: eighth/c/1.c
#include <stdio.h>

float calcularPrestacaoAtraso(float valor, float taxa, int tempo) {
    float prestacao = valor + (valor * (taxa / 100) * tempo);
    return prestacao;
}

int main() {
    float valor = 1000;
    float taxa = 5;
    int tempo = 2;
    float prestacaoAtraso = calcularPrestacaoAtraso(valor, taxa, tempo);
    printf("O valor da prestação em atraso é: R$ %.2f\n", prestacaoAtraso);
    return 0;
}

File: eighth/python/1.py
def calcular_prestacao_atraso(valor, taxa, tempo):
    prestacao = valor + (valor * (taxa / 100) * tempo)
    return prestacao

# Exemplo de uso
valor = 1000
taxa = 5
tempo = 2
prestacao_atraso = calcular_prestacao_atraso(valor, taxa, tempo)
print(f"O valor da prestação em atraso é: R$ {prestacao_atraso:.2f}")
